qiskit/test/mock/fake_openpulse_3q.py

```diff
@@ -116,12 +116,17 @@ def __init__(self):
                 'u0': {
                     'type': 'control',
                     'purpose': 'cross-resonance',
-                   'operates': {'qubits': [0, 1]}
+                    'operates': {'qubits': [0, 1]}
                 },
                 'u1': {
                     'type': 'control',
                     'purpose': 'cross-resonance',
                     'operates': {'qubits': [1, 0]}
+                },
+                'u2': {
+                    'type': 'control',
+                    'purpose': 'cross-resonance',
+                    'operates': {'qubits': [2]}
```

Comment:
This should connect two qubits, for example [0,2], [2, 0], [1,2] and [2, 1] are all valid choices.

Reply:
Sorry, fixed in [4a03749](https://github.com/Qiskit/qiskit-terra/pull/4085/commits/4a03749b9cdff8f900f35065cb9e6169a1fca985)